### PR TITLE
Allow selected zone to appear above state layer

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
The selected zone is not clearly visible from the new state border layer

## Description
This PR adds a new layer to the map that is transparent unless the zone is selected. 

The state border layer is positioned behind this new layer.

Also update the selected zone border thickness to add clarity to the selected zone. 
### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Comment by tonypls** [comment]
@Alportan I've request you as a review to get some feedback on this change. You can view the branch deployment above, feel free to give some feedback here or we could do in person 🙏

File: web/src/features/map/map-layers/ZonesLayer.tsx [hunk 1]
```diff
@@ -39,7 +39,22 @@ export default function ZonesLayer() {
...
+        'fill-color': [
+          'coalesce',
+          ['feature-state', 'color'],
+          ['get', 'color'],
+          theme.clickableFill,
+        ],
+        'fill-opacity': [
+          'case',
+          ['boolean', ['feature-state', 'selected'], false],
+          0.8,
```
Comment (VIKTORVAV99) [inline web/src/features/map/map-layers/ZonesLayer.tsx:52]: Maybe try a little smaller value here, perhaps 0.7.

**Review by VIKTORVAV99 — APPROVED** [review]
Working as expected so approving, I have a small suggestion you can try if you want. Might just be my own personal preference but I'd like to see the zone state borders a bit more clearly.

File: web/src/features/map/map-layers/ZonesLayer.tsx (hunk 1 above)
Comment (tonypls) [reply web/src/features/map/map-layers/ZonesLayer.tsx:52]: Yeah I think that's a good balance, will update now

**Review by Alportan — APPROVED** [review]
Nothing to add besides the fact that it looks good! Approved on my side! 💪

**Review by madsnedergaard — APPROVED** [review]
Nice and simple ❤️

**Pull request merged**